packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -352,7 +352,10 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
                     items,
                     totalItemCount,
                     hasCheckmarks,
-                    hasIcons
+                    hasIcons,
+                    defaultMenuItemRenderer: (_items, _index, _focusableElementIndex, _totalItemCount, _hasCheckmarks, _hasIcons) => {
```

Comment:
Inline lambda's can add a perf hit. You should be able to just pass in `this._renderMenuItem` directly to the defaultMenuItemRenderer prop.

Reply:
The default item renderer accesses the props through `this` so we have to capture it here. I think this means we need the lambda?

Reply:
Discussed offline - will change _renderMenuItem to be an arrow function in order to capture this.